.github/ISSUE_TEMPLATE.md

```diff
@@ -1,33 +1,36 @@
-**In raising this issue, I confirm the following (please check boxes, eg [X]) Failure to fill the template will close your issue:**
+**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`
 
 - [] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
-- [] The issue I am reporting can be *replicated*
+- [] The issue I am reporting can be *replicated*.
 - [] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).
 
-**How familiar are you with the codebase?:**
+**How familiar are you with the the source code relevant to this issue?:**
 
-_{replace this text with a number from 1 to 10, with 1 being not familiar, and 10 being very familiar}_
+`{Replace this with a number from 1 to 10. 1 being not familiar, and 10 being very familiar}`
 
 ---
-**[BUG REPORT | OTHER]:**
+**Expected behaviour:**
 
-Please [submit your feature request here](https://discourse.pi-hole.net/c/feature-requests), so it is votable by the community.  It's also easier for us to track.
```

Comment:
How about something more along the lines of:

**Feature Request**
- If this is a feature request, please do not submit this as an issue here. Instead post it to [our discourse forums](https://discourse.pi-hole.net/c/feature-requests) so that it can be better tracked, and voted on by the community